AudioPlayerRecorderSamples/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,8 @@
     <string name="hello_world">Hello world!</string>
     <string name="action_settings">Settings</string>
 
+    <string name="permission_rationale">You need to grant application record audio permission in Settings application.</string>
+    <string name="permission_action_allow">Allow</string>
+    <string name="permission_action_settings">Settings</string>
+    <string name="error_storage_not_available">Storage not available.</string>
```

Comment:
Also, after using the sample, I think we should remove `<string name="permission_action_allow">Allow</string>` and just use "OK".

From a UX perspective, the users have 2 subsequent interactions where they press "Allow". In the SnackBar, it seems like the tap will allow the microphone usage, but it doesn't. So I would change it to "OK" like in Google's sample, given they are requested for the permission right after it.